src/Libraries/RevitNodes/Elements/FamilyType.cs

```diff
@@ -60,6 +65,29 @@ private void InitFamilySymbol(Autodesk.Revit.DB.FamilySymbol symbol)
             InternalSetFamilySymbol(symbol);
         }
 
+        /// <summary>
+        /// Get Solid from Element helper method
+        /// </summary>
+        /// <param name="element"></param>
+        /// <returns></returns>
+        private static Solid GetSolidFromElement(Autodesk.Revit.DB.Element element)
```

Comment:
There is a property method called Element.Solids. Is there any specific reason that we need another method for ElementType?